phantomgames/GameDetails/GameDetailViewController.swift

```diff
@@ -83,6 +87,14 @@ class GameDetailViewController: UIViewController {
             gameButton.isEnabled = false
         }
     }
+    private func updateFavoriteButton() {
+        if gameDetailViewModel.isFavorite {
```

Comment:
let image =  gameDetailViewModel.isFavorite  ? UIImage(systemName: "star.fill") : UIImage(systemName: "star"),
 favoriteButton.setImage(image, for: .normal)